internal/importers/base/base_importer.go

```diff
@@ -13,30 +13,30 @@ import (
 	"github.com/pkg/errors"
 )
 
-// baseImporter provides a common implementation for importing from other
+// Importer provides a common implementation for importing from other
 // dependency managers.
-type baseImporter struct {
-	logger   *log.Logger
-	verbose  bool
-	sm       gps.SourceManager
-	manifest *dep.Manifest
-	lock     *dep.Lock
+type Importer struct {
+	Logger   *log.Logger
+	Verbose  bool
+	Sm       gps.SourceManager
```

Comment:
Let's make this `SourceManager` instead. Need to behave ourselves when exporting 😄